src/webots/control/WbController.cpp

```diff
@@ -251,9 +251,13 @@ void WbController::start() {
         warn(tr("Environment variables from runtime.ini could not be loaded: the file contains illegal definitions."));
       else {
         for (int i = 0; i < iniParser.size(); ++i) {
-          if (iniParser.sectionAt(i).compare("environment variables with relative paths") != 0 &&
-              iniParser.sectionAt(i).compare("environment variables for Linux") != 0 &&
-              iniParser.sectionAt(i).compare("environment variables for Linux 64") != 0)
+          if (iniParser.sectionAt(i) == "environment variables with relative paths")
```

Comment:
On Linux when `WEBOTS_FIREJAIL_CONTROLLERS` is set, the warning is probably displayed twice, once here, and once [there](https://github.com/cyberbotics/webots/blob/8208b5010f88b8bd1144011faf753f59557bef10/src/webots/control/WbController.cpp#L375-L377). Isn't it?